### PR TITLE
fixed broken links

### PR DESCRIPTION
**Pull request by leandrojk**
# Pull Request Checklist

* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Typesafe CLA](https://www.typesafe.com/contribute/cla)?
* [x] Have you [squashed your commits](https://www.playframework.com/documentation/latest/WorkingWithGit#Squashing-commits)?
* [ ] Have you added copyright headers to new files?
* [x] Have you updated the documentation?
* [ ] Have you added tests for any changed functionality?

## Purpose

Fixex broken links in one documentation file.

### CODE REVIEW

File: docs/manual/working/scalaGuide/main/json/ScalaJson.md [hunk 1]
```diff
@@ -38,24 +38,24 @@ The [`play.api.libs.json`](api/libs/json/) package contains data structures for
...
+- [`JsNumber`](api/scala/play/api/libs/jsonJsNumber.html)
+- [`JsBoolean`](api/scala/play/api/libs/jsonJsBoolean.html)
+- [`JsObject`](api/scala/play/api/libs/jsonJsObject.html)
+- [`JsArray`](api/scala/play/api/libs/jsonJsArray.html)
+- [`JsNull`](api/scala/play/api/libs/jsonJsNull$.html)
 
 Using the various `JsValue` types, you can construct a representation of any JSON structure.
 
-### [`Json`](api/libs/json/Json$.html)
+### [`Json`](api/scala/play/api/libs/jsonJson$.html)
```
Comment (gmethvin) [inline docs/manual/working/scalaGuide/main/json/ScalaJson.md]: I think you forgot a `/`: `api/scala/play/api/libs/json/Json$.html` (same issue elsewhere in this file).

Comment (leandrojk) [reply docs/manual/working/scalaGuide/main/json/ScalaJson.md]: Sorry! I`ll fix that.

File: docs/manual/working/scalaGuide/main/json/ScalaJson.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
 # JSON basics
 
-Modern web applications often need to parse and generate data in the JSON (JavaScript Object Notation) format. Play supports this via its [JSON library](api/libs/json/).
+Modern web applications often need to parse and generate data in the JSON (JavaScript Object Notation) format. Play supports this via its [JSON library](api/scala/play/api/libs/json).
```
Comment (gmethvin) [inline docs/manual/working/scalaGuide/main/json/ScalaJson.md]: I'm not sure this link is what we want. I think we want to go to https://www.playframework.com/documentation/2.6.x/api/scala/play/api/libs/json/package.html, so the correct link should be `api/scala/play/api/libs/json/package.html`

Comment (leandrojk) [reply docs/manual/working/scalaGuide/main/json/ScalaJson.md]: You are right. The link for 2.5.x version is https://www.playframework.com/documentation/2.5.x/api/scala/play/api/libs/json/package.html

**Comment by gmethvin** [comment]
2.6.x: 6d832ca

**Pull request merged**